packages/@o3r/eslint-config-otter/schematics/ng-add/linter/index.ts

```diff
@@ -25,14 +24,14 @@ export function updateLinterConfigs(options: { projectName: string | null }, roo
     const eslintExists = tree.exists(eslintFilePath);
 
     if (eslintExists) {
-      const eslintFile = commentJson.parse(tree.read(eslintFilePath)!.toString()) as { extends?: string | string[] };
+      const eslintFile = tree.readJson(eslintFilePath) as { extends?: string | string[] };
```

Comment:
can't eslint file be a JS file ?

Reply:
It can and in this case the previous code was not working neither :S

Reply:
Maybe we could open an issue to track it and fix it later

Reply:
Reported to issue #588 .